src/containers/Results/Results.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+import './Results.css'
+// import * as actions from '../../actions';
+import { connect } from 'react-redux'
+import { Link } from "react-router-dom";
+
+const Results = ({solarData}) => {
+    const [savingsValue] = useState(50)
+    const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December']
+
+    const tableData = months.map((item, index)=>{
+        return (
```

Comment:
To fix the indentation on this file I can go into it and save so that it auto formats it for us.